cmd/ddev/cmd/scripts/test_ddev.sh

```diff
@@ -50,6 +50,11 @@ ddev debug configyaml | grep -v web_environment
 header "existing project customizations"
 grep -r -L "#ddev-generated" .ddev/docker-compose.*.yaml .ddev/php .ddev/mutagen .ddev/apache .ddev/nginx* .ddev/*-build .ddev/mysql .ddev/postgres 2>/dev/null
```

Comment:
But there is already `.ddev/nginx*`.

Run:
```
mkdir -p .ddev/nginx && touch .ddev/nginx/custom.conf
grep -r -L "#ddev-generated" .ddev/docker-compose.*.yaml .ddev/php .ddev/mutagen .ddev/apache .ddev/nginx* .ddev/*-build .ddev/mysql .ddev/postgres 2>/dev/null
```
Result:
```
.ddev/nginx/custom.conf
```

I think the problem may be that the user had `#ddev-generated` in their file.